packages/x-charts-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-beta.5",
+  "version": "7.0.0-beta.6",
```

Comment:
Do we have plans to ship stable in the near future? 🤔

Reply:
I assumed that in https://github.com/mui/mui-x/pull/15104 🤷‍♂️

Reply:
> Or you mean shipping stable under v7 major?

Yes, that was my question to Alex. 🤔 
We did move Pickers to stable before v6. 🤔